### ISSUE TITLE
Dlang (gdc) is not in list of packages

### ISSUE THREAD

**Issue by Siemargl** [msys2/MINGW-packages#6806]
Arising with gcc 9.2 D language (dlang.org) still not supported in msys

**Comment by lazka** [comment]
Do you know what would be needed for this?

**Comment by Siemargl** [comment]
Please explain.

I dont think, that its too complex, because Dlang and its libraries are cross-platform.

**Comment by revelator** [comment]
possible to build gdc i just did that, but libphobos is a nogo without a lot of patching complains about undefined FILE so it seems 
stdio.d does not have the correct interface for exporting this symbol with a mingw-w64 compiler. can still use gdc but need to use the flag to turn of linking to libphobos by default.

**Comment by Geod24** [comment]
Since LDC works with Mingw, one could probably look into upstreaming the changes they made.
See e.g. https://github.com/ldc-developers/phobos/blob/c6b1d8354dcbf741e8544aefe3e05fd69a927b31/std/stdio.d#L40-L43

**Comment by kinke** [comment]
Nope, LDC doesn't work with MinGW. There were some attempts to get it working, but I was never really interested in getting it done.
